pkg/datastore/deploymentchainstore.go

```diff
@@ -183,3 +184,13 @@ func (s *deploymentChainStore) UpdateDeploymentChain(ctx context.Context, id str
 		return dc.Validate()
 	})
 }
+
+func (s *deploymentChainStore) AddDeploymentToBlock(ctx context.Context, chainID string, deployment *model.Deployment) error {
```

Comment:
How about `UpdateNodeDeployment`

Reply:
Update is a bit hard to get what it is, lets me rename it as `AddNodeDeployment`

Reply:
Sure.